src/status_im/contexts/settings/advanced/view.cljs

```diff
@@ -0,0 +1,266 @@
+(ns status-im.contexts.settings.advanced.view
+  (:require [clojure.string :as string]
+            [legacy.status-im.fleet.core :as fleets]
+            [quo.core :as quo]
+            [react-native.clipboard :as clipboard]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.events-helper :as events.helper]
+            [status-im.config :as config]
+            [status-im.contexts.settings.advanced.events]
+            [status-im.contexts.settings.advanced.style :as style]
+            [status-im.feature-flags :as ff]
+            [utils.datetime :as datetime]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- static-header
+  []
+  [rn/view {:style style/header}
+   [quo/page-nav
+    {:background :blur
+     :icon-name  :i/arrow-left
+     :on-press   events.helper/navigate-back}]])
+
+(defn- advanced-item
+  [{:keys [label data] :as _item-data}]
+  (let [customization-color (rf/sub [:profile/customization-color])]
+    [quo/category
+     {:container-style     style/advanced-item
+      :label               label
+      :customization-color customization-color
+      :data                data
+      :blur?               true
+      :list-type           :settings}]))
+
+(defn- format-timestamp
+  [prefix-string last-backup-timestamp]
+  (-> (str prefix-string " " (datetime/timestamp->relative (* 1000 last-backup-timestamp)))
+      (string/capitalize)
+      (string/replace #"am|pm" {"am" "AM" "pm" "PM"})))
+
+(defn- waku-backup-settings
+  []
+  (let [{:keys [backup-enabled?
+                last-backup]} (rf/sub [:profile/profile])
+        performing-backup?    (rf/sub [:backup/performing-backup])
+        customization-color   (rf/sub [:profile/customization-color])]
+    [:<>
+     [quo/drawer-top {:title (i18n/label :t/waku-backup)}]
+     (comment
```

Comment:
Is this a leftover?

Reply:
Nope, this code is useful to complete the UI with an explanation on what enabling the backup implies. I'd prefer to keep it and just adjust it in a follow up where all the settings can provide an explanation on what they do

Reply:
If you are sure you'll be able to work on the follow-up soon(ish) that's okay, otherwise it'd be a good to explain this as a TODO, or even better have an issue describing this follow-up work to help us more easily manage the project.